src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    public static void printTask(Task newTask, int taskCount) {
+        String toPrint = newTask.toString();
+        System.out.println("Ok! I've added this task:");
+        System.out.println(newTask);
+        System.out.println("Now you have " + taskCount + " tasks in your list uwu");
+    }
+
+    public static void printTaskList(Task[] list, int totalTasks) {
+        Task[] printList = Arrays.copyOf(list, totalTasks); // only copy until the part you want
+        int i = 1;
+        System.out.println("Here are the tasks in your list:");
+        for (Task task : printList) {
+            System.out.println(i + "." + task);
+            i += 1;
+        }
+    }
+
+    public static void markDone(Task doneTask, int taskCount, Task[] taskList) {
+        String n = doneTask.description.substring(5);
```

Comment:
Is there a meaning behind the number 5? Do consider creating a variable which explains its significance.